packages/core/utils/src/relativeBundlePath.js

```diff
@@ -1,13 +1,19 @@
-// @flow
+// @flow strict-local
+
 import type {Bundle} from '@parcel/types';
+
 import path from 'path';
 import nullthrows from 'nullthrows';
 
-export function relativeBundlePath(from: Bundle, to: Bundle) {
+export function relativeBundlePath(
+  from: Bundle,
+  to: Bundle,
+  opts: {|leadingDotSlash: boolean|} = {leadingDotSlash: true},
+) {
   let p = path
     .relative(path.dirname(nullthrows(from.filePath)), nullthrows(to.filePath))
     .replace(/\\/g, '/');
-  if (p[0] !== '.') {
+  if (opts.leadingDotSlash && p[0] !== '.') {
```

Comment:
@devongovett is this the best place for this?

Reply:
seems fine...